docs/docsite/rst/dev_guide/testing_units_modules.rst

```diff
@@ -332,7 +332,7 @@ testing for the correct exception::
         })
 
         with self.assertRaises(AnsibleExitJson) as result:
-            my_module.main()
+            self.module.main()
```

Comment:
The examples in this doc are inconsistent. Some are class based, and some are not. We're encouraging moving away from classes in tests, so I'm not sure we should be changing this. Instead, I'd rather see the rest of the examples updated.